### PR TITLE
1238 feature create react lib setup

### LINKED ISSUE #1238: [Feature?]: Create React lib/NPM setup
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

Prep the workspace for the new lib and setup NPM stuff.

### Examples 🌈

_No response_

### Motivation 🔦

React, NPM, Versioning

### PR DESCRIPTION
**Pull request by caseybaggz**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
closes #1238 

## What is the new behavior?
- Adds new `react` workspace and project setup
- Adds `react-sandbox` internal workspace
- Updates versions.mjs to include "react" workspace
- kills unnecessary entry files in HS/React-utils

## Other information
https://codesandbox.io/s/ps-react-typescript-forked-ugwsw5?file=/src/App.tsx

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+erDiagram
+  PANDO ||--o{ DESIGN-TOKENS : contains
+  PANDO ||--o{ ICONS : contains
+  PANDO ||--o{ HEADLESS-STYLES : contains
+  PANDO ||--o{ REACT : contains
+  PANDO ||--o{ REACT-UTILS : contains
+  DESIGN-TOKENS ||..|{ NORMALIZE-SETUP : contains
+  NORMALIZE-SETUP ||..|{ THEMES : contains
+  HEADLESS-STYLES ||--|{ DESIGN-TOKENS : uses
+  REACT ||--|{ HEADLESS-STYLES : uses
```
Comment (Rykus0) [inline packages/react/README.md:17]: Should we add that React also uses react-utils and icons?

File: packages/react/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This package utilizes the latest version of React to deliver component-based abstractions, helpers, and hooks of the Headless-styles library.
+
+### React General Function
+
+The react package is dependant on the Headless-styles library, and works cohesively with the other libraries when combined with them. It's true purpose is to provide a component and hook library within the Pando Normalize system (i.e. themes, etc.).
+
+## Install
+
+This project uses Yarn 3 with PnP so there are no setup commands needed. If you get any errors, you may need to run an initial `yarn install` or esnure you are using Node >= 18.
```
Comment (Rykus0) [inline packages/react/README.md:28]:
```suggestion
This project uses Yarn 3 with PnP so there are no setup commands needed. If you get any errors, you may need to run an initial `yarn install` or ensure you are using Node >= 18.
```

File: packages/react/package.json [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    "pluralsight react utils",
+    "react",
+    "utils",
+    "react-utils",
+    "custom hooks",
+    "hooks",
+    "react hooks",
+    "components",
+    "ui components",
+    "pluralsight"
```
Comment (Rykus0) [inline packages/react/package.json:96]:
```suggestion
    "ui components"
```

File: packages/react/README.md (hunk 1 above)
Comment (caseybaggz) [reply packages/react/README.md:17]: React utils, yes - but it will not use Icons. Remember these are composable components, not all in one solutions.

Comment (caseybaggz) [reply packages/react/README.md:17]: Go ahead and add a suggestion and I'll get that one in.

Comment (Rykus0) [reply packages/react/README.md:17]:
```suggestion
  REACT ||--|{ HEADLESS-STYLES : uses
  REACT ||--|{ REACT-UTILS : uses
```

**Review by Rykus0 — APPROVED** [review]

**Pull request merged**